src/main/java/gyro/aws/eks/EksNodegroupResource.java

```diff
@@ -342,6 +361,16 @@ public void copyFrom(Nodegroup model) {
             specification.copyFrom(model.launchTemplate());
             setLaunchTemplateSpecification(specification);
         }
+
+        if (model.hasTaints()) {
```

Comment:
We empty the list in case there are any discrepancies with the state file.
```suggestion
        getTaint().clear();
        if (model.hasTaints()) {
```